server.js

```diff
@@ -282,6 +282,13 @@ module.exports = function(options) {
         auth: "stripe"
       },
       handler: routes['stripe-dispute']
+    }, {
+      method: "POST",
+      path: "/stripe/charge-succeeded",
```

Comment:
I wonder if it's wise to put webhook routes behind something like `webhook/stripe/charge-succeeded/dispute-callback/etc`?